astro-basic-ts/src/components/MyButton.tsx

```diff
@@ -0,0 +1,17 @@
+import { Button } from "primereact/button"
+import "primereact/resources/themes/lara-light-indigo/theme.css";
+import "primereact/resources/primereact.min.css";
+import "primeicons/primeicons.css";
```

Comment:
My only thought was i wonder if these imports should go int Layout so they are used through the whole app?

Reply:
Well, it depends...if you are not using anything PrimeReact related on the homepage, then there is no reason to load the icons font, and a relatively large CSS file that does not go through tree shaking to remove unused classes.

I could probably move the imports into the PrimeReact page if that makes more sense. Let me know.

Reply:
yep that would make more sense for someone looking at this